src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorPrimitives.netcore.cs

```diff
@@ -9739,15 +9812,210 @@ public static Vector512<T> Invoke(Vector512<T> x, Vector512<T> y)
         /// <summary>x / y</summary>
         internal readonly struct DivideOperator<T> : IBinaryOperator<T> where T : IDivisionOperators<T, T, T>
         {
+            public static bool Vectorizable => true;
             public static T Invoke(T x, T y) => x / y;
             public static Vector128<T> Invoke(Vector128<T> x, Vector128<T> y) => x / y;
             public static Vector256<T> Invoke(Vector256<T> x, Vector256<T> y) => x / y;
             public static Vector512<T> Invoke(Vector512<T> x, Vector512<T> y) => x / y;
         }
 
+        /// <summary>y / x</summary>
+        internal readonly struct InvertedDivideOperator<T> : IBinaryOperator<T> where T : IDivisionOperators<T, T, T>
+        {
+            public static bool Vectorizable => true;
+            public static T Invoke(T x, T y) => y / x;
+            public static Vector128<T> Invoke(Vector128<T> x, Vector128<T> y) => y / x;
+            public static Vector256<T> Invoke(Vector256<T> x, Vector256<T> y) => y / x;
+            public static Vector512<T> Invoke(Vector512<T> x, Vector512<T> y) => y / x;
+        }
+
+        /// <summary>T.Ieee754Remainder(x, y)</summary>
+        internal readonly struct Ieee754RemainderOperator<T> : IBinaryOperator<T> where T : IFloatingPointIeee754<T>
+        {
+            public static bool Vectorizable => false;
+            public static T Invoke(T x, T y) => T.Ieee754Remainder(x, y);
+            public static Vector128<T> Invoke(Vector128<T> x, Vector128<T> y) => throw new NotSupportedException();
+            public static Vector256<T> Invoke(Vector256<T> x, Vector256<T> y) => throw new NotSupportedException();
+            public static Vector512<T> Invoke(Vector512<T> x, Vector512<T> y) => throw new NotSupportedException();
+        }
+
+        // Ieee754Remainder
+
+        internal readonly struct ReciprocalOperator<T> : IUnaryOperator<T> where T : IFloatingPoint<T>
```

Comment:
Do we want to propose exposing this on `IFloatingPoint<T>` for parity? Same for `ReciprocalSqrt`?

Reply:
Would it achieve anything on `IFloatingPoint<T>` you couldn't just do with `1 / value`?  Seems like it doesn't add significant enough value there.